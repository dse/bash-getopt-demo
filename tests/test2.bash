#!/usr/bin/evn bash

exec 2>&1

# NOTE: earlier versions of Bash treat the following differently:
# - $'\c\\'
# - $'\uxxxx'
# - $'\Uxxxxxxxx'

bin/bash-getopt-demo $'\c\\' $'\u00a1' $'\u2014' $'\U0001F4A9'
