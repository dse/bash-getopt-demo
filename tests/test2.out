---
This is the program:
  bin/bash-getopt-demo
invoked with the following arguments:
  $'\034' Â¡ â€” ðŸ’©
verbosity level is 0
one=
two=
three=
no option arguments
4 remaining positional arguments:
   1. $'\034'
   2. Â¡
   3. â€”
   4. ðŸ’©
